illuminate/view/CVE-2021-43808.yaml

```diff
@@ -0,0 +1,14 @@
+title:     Possible cross-site scripting (XSS) vulnerability in the Blade templating engine
+link:      https://github.com/laravel/framework/security/advisories/GHSA-66hf-2p6w-jqfw
+cve:       CVE-2021-43808
+branches:
+  "6.x":
+    time:     ~
+    versions: ['>=6.0.0', '<6.20.42']
```

Comment:
What about versions below 6.0? Are they not affected?

Reply:
The github advisory marks them as affected.

Reply:
this means that the lower bound should be removed here